censys/certificates.py

```diff
@@ -6,6 +6,25 @@
 class CensysCertificates(CensysIndex):
 
     INDEX_NAME = "certificates"
+    MAX_PER_BULK_REQUEST = 50
+
+    def __init__(self, *args, **kwargs):
+        CensysIndex.__init__(self, *args, **kwargs)
+        self.bulk_path = "/bulk/%s" % self.INDEX_NAME
```

Comment:
Use `.format()` for Python 3 compatibility

Reply:
Done! I do not have merge permissions though